### PR TITLE
Dcos 47967 fix license broken links

### PR DESCRIPTION
**Pull request by rathohit311356**
DCOS-47967

### CODE REVIEW

File: repo/packages/K/kafka-zookeeper/0/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "zookeeper",
     "kafka"
   ],
-  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nBy using this service you agree to the license terms here: https://github.com/mesosphere/dcos-zookeeper/blob/master/LICENSE\n\nVersions: Apache ZooKeeper 3.4.11",
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nVersions: Apache ZooKeeper {{zookeeper-version}}",
```
Comment (zmalik) [inline repo/packages/K/kafka-zookeeper/0/package.json]: @rathohit311356 you have tested this? 
{{zookeeper-version}}

**Review by zmalik — CHANGES_REQUESTED** [review]

File: repo/packages/K/kafka-zookeeper/0/package.json (hunk 1 above)
Comment (zmalik) [reply repo/packages/K/kafka-zookeeper/0/package.json]: please change it back to the version numbers as it was before.

File: repo/packages/K/kafka-zookeeper/100/package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "zookeeper",
     "kafka"
   ],
-  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\n\tBy using this service you agree to the license terms here: https://github.com/mesosphere/dcos-zookeeper/blob/master/LICENSE\n\nVersions: Apache ZooKeeper 3.4.11",
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nVersions: Apache ZooKeeper {{zookeeper-version}}",
```
Comment (zmalik) [inline repo/packages/K/kafka-zookeeper/100/package.json]: same here, use the version

File: repo/packages/K/kafka-zookeeper/200/package.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     "zookeeper",
     "kafka"
   ],
-  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\n\tBy using this service you agree to the license terms here: https://github.com/mesosphere/dcos-zookeeper/blob/master/LICENSE\n\nVersions: Apache ZooKeeper 3.4.12",
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nVersions: Apache ZooKeeper {{zookeeper-version}}",
```
Comment (zmalik) [inline repo/packages/K/kafka-zookeeper/200/package.json]: and here ^

File: repo/packages/K/kafka-zookeeper/300/package.json [hunk 4]
```diff
@@ -17,7 +17,7 @@
     "zookeeper",
     "kafka"
   ],
-  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\n\tBy using this service you agree to the license terms here: https://github.com/mesosphere/dcos-zookeeper/blob/master/LICENSE\n\nVersions: Apache ZooKeeper 3.4.13",
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nVersions: Apache ZooKeeper {{zookeeper-version}}",
```
Comment (zmalik) [inline repo/packages/K/kafka-zookeeper/300/package.json]: 3.4.13

File: repo/packages/K/kafka-zookeeper/400/package.json [hunk 5]
```diff
@@ -17,7 +17,7 @@
     "zookeeper",
     "kafka"
   ],
-  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\n\tBy using this service you agree to the license terms here: https://github.com/mesosphere/dcos-zookeeper/blob/master/LICENSE\n\nVersions: Apache ZooKeeper 3.4.13",
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nVersions: Apache ZooKeeper {{zookeeper-version}}",
```
Comment (zmalik) [inline repo/packages/K/kafka-zookeeper/400/package.json]: use 3.4.13 also here

**Review by zmalik — CHANGES_REQUESTED** [review]

File: repo/packages/K/kafka-zookeeper/0/package.json (hunk 1 above)
Comment (rathohit311356) [reply repo/packages/K/kafka-zookeeper/0/package.json]: Changes Done

File: repo/packages/K/kafka-zookeeper/100/package.json (hunk 2 above)
Comment (rathohit311356) [reply repo/packages/K/kafka-zookeeper/100/package.json]: Done

File: repo/packages/K/kafka-zookeeper/200/package.json (hunk 3 above)
Comment (rathohit311356) [reply repo/packages/K/kafka-zookeeper/200/package.json]: Done

File: repo/packages/K/kafka-zookeeper/300/package.json (hunk 4 above)
Comment (rathohit311356) [reply repo/packages/K/kafka-zookeeper/300/package.json]: Done

File: repo/packages/K/kafka-zookeeper/400/package.json (hunk 5 above)
Comment (rathohit311356) [reply repo/packages/K/kafka-zookeeper/400/package.json]: Done

File: repo/packages/K/kafka-zookeeper/300/package.json [hunk 6]
```diff
@@ -17,7 +17,7 @@
     "zookeeper",
     "kafka"
   ],
-  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\n\tBy using this service you agree to the license terms here: https://github.com/mesosphere/dcos-zookeeper/blob/master/LICENSE\n\nVersions: Apache ZooKeeper 3.4.13",
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 4000 MB MEM | 5000 MB Disk\n\nVersions: Apache ZooKeeper 3.14.13",
```
Comment (zmalik) [inline repo/packages/K/kafka-zookeeper/300/package.json]: typo here, should be `3.4.13`

**Review by zmalik — CHANGES_REQUESTED** [review]

File: repo/packages/K/kafka-zookeeper/300/package.json (hunk 6 above)
Comment (rathohit311356) [reply repo/packages/K/kafka-zookeeper/300/package.json]: Corrected

**Review by zmalik — APPROVED** [review]
LGTM 
let's wait for 1 more review here

**Review by vishnu2kmohan — APPROVED** [review]
LGTM, thanks @rathohit311356 and @zmalik

**Pull request merged**